addons/danger/functions/fnc_brainEngage.sqf

```diff
@@ -61,6 +61,11 @@ if (
     _timeout + 4
 };
 
+// running are busy and shouldn't suppress
+if ((speed _unit) isEqualTo 0) exitWith {
```

Comment:
Interesting. My intention was to run this as _isNotEqualTo_, yet this worked fine.  Will need a bit more testing :/